### PR TITLE
Add post build events to VS Projects to copy output into bin directory.

### PR DESCRIPTION
**Pull request by ZehMatt**
This will put the OpenTTD executable in the bin folder after successfully building the project.

### CODE REVIEW

**Comment by TrueBrain** [comment]
Seems inconsistent: sometimes `..\bin\`, sometimes `..\bin\$(ConfigurationName)`, depending on the MSVC project. Is that a deliberate choice?

Also, does the binary start after compiling? Relative to the binary normally should be the language files, and I don't see those changed, so I wonder if it still works?

@glx22: can you check if this doesn't break anything? Tnx!

**Comment by ZehMatt** [comment]
Sorry forgot to update them.

**Comment by ZehMatt** [comment]
And yes the binary starts fine from within the bin directory. The bin folder contains following:
```
<DIR> ai
<DIR> baseset
<DIR> game
<DIR> lang
<DIR> scripts
<BIN> openttd.exe
<BIN> openttd.iobj
<BIN> openttd.ipdb
<BIN> openttd.pdb
```

**Comment by glx22** [comment]
Outputting to bin seems ok, but it will complicate installer creation. Currently it's just a matter of running mingw, msvc for 32bit, msvc for 64bit, then os/windows/installer/build_installers.bat does the magic using clearly defined locations to get the right openttd.exe

**Comment by TrueBrain** [comment]
Guess that can now be simplified? But as it was written this explicit, we might want to test etc, if everything still works.

**Comment by ZehMatt** [comment]
Is this something I could adjust to keep the installer working?

**Comment by TrueBrain** [comment]
If you wouldn't mind, please do. If possible it should be part of this PR, as breaking the installer is a bit evil :D I am sure if you check the files in the folder @glx22 mentioned you can figure it out :)

**Comment by ZehMatt** [comment]
I had to slightly change the approach since the installer requires for each configuration the binary. So instead of changing the output directory I've added a post build event in each VS project which copies the binary into the bin directory. This way I don't break anything and it should be slightly more comfortable.

File: projects/openttd_vs100.vcxproj [hunk 1]
```diff
@@ -148,6 +148,9 @@
     <Manifest>
       <AdditionalManifestFiles>dpi_aware.manifest;os_versions.manifest</AdditionalManifestFiles>
     </Manifest>
+	<PostBuildEvent>
```
Comment (LordAro) [inline projects/openttd_vs100.vcxproj]: indentation needs checking

**Comment by glx22** [comment]
It's indeed better that way, similar to what is done for MinGW (and other platforms). Maybe it could be a good idea to also modify os/windows/installer/version_win9x.txt to pick openttd.exe in 'objs/release' instead of 'bin', for consistency.

**Comment by ZehMatt** [comment]
Could you tell me which project produces the win9x binary? I'm not exactly sure which path to pick here since all projects put it to objs\win32\Release.

**Comment by glx22** [comment]
win9x binary is generated with MinGW, so it uses Makefile and gcc. The output goes in objs/release (objs/debug for debug builds of course). So it's just a small change in os/windows/installer/version_win9x.txt from ${PATH_ROOT)bin to ${PATH_ROOT}objs\release.

**Review by glx22 — APPROVED** [review]

**Pull request merged**